UKHomeOffice/sgmr-service
# SPCR-146 Change PortField component to match new ports response structure

## Description
This PR changes remove the extra `data` property when storing the response from the `ports` endpoint. The additional `data` was a feature of the refdata service which we are no longer using to search for ports. This PR also adds an initial test for the `PortField` component testing only the functionality I have added.

## To test
- Pull and run against dev instance
- Sign in and create new voyage plan
- On the departure page, enter a string i.e. Portsm
- *You should see a list of ports returned relating to the query you just made*
- Navigate to the arrival page and do the same thing (try with varying queries)

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
